### PR TITLE
m_pause OK

### PR DESCRIPTION
**Pull request by Prakxo**

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

File: src/code/zurumode.c [hunk 1]
```diff
@@ -185,7 +186,7 @@ void zurumode_callback(void* p) {
         zerucheck_init(&zuruKeyCheck);
         APPNMI_ZURUMODE_FLP();
         zurumode_update();
-        if ((mgr->pads[0].button) & 0x80) {
+        if ((mgr->pads[0].button) & RESET_BUTTON) {
```
Comment (hensldm) [inline src/code/zurumode.c:189]:
```suggestion
        if (CHECK_BTN_ALL(mgr->pads[0].button, RESET_BUTTON)) {
```

File: src/code/m_pause.c [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        pause->enabled = !pause->enabled;
+    }
+
+    if ((!pause->enabled) || (CHECK_BTN_ALL(input->cur.button, Z_TRIG) &&
+                              (CHECK_BTN_ALL(input->press.button, R_TRIG) ||
+                               (CHECK_BTN_ALL(input->cur.button, R_TRIG) && (++pause->timer >= 9))))) {
+        pause->timer = 0;
+        return 1;
+    }
+    return 0;
```
Comment (hensldm) [inline src/code/m_pause.c:20]:
```suggestion
        return true;
    }
    return false;
```

File: src/code/zurumode.c (hunk 1 above)
Comment (Prakxo) [reply src/code/zurumode.c:189]: Does that still match? 🤔 The macro looks very different than what is here

Comment (Prakxo) [reply src/code/zurumode.c:189]: Macro doesn't match, so it can't be used here

Comment (hensldm) [reply src/code/zurumode.c:189]:
```suggestion
        if (mgr->pads[0].button & RESET_BUTTON) {
```
Ah dang. Than I would suggest just removing the extra parens.

**Review by hensldm — APPROVED** [review]

**Pull request merged**